public/managehashes.php

```diff
@@ -88,14 +89,19 @@ function UnlinkHash(user, gameID, hash, elem) {
         <?php
         echo gameAvatar($gameData, iconSize: 64);
 
+        echo "<div class='mt-2 flex flex-col gap-1'>";
+        echo " <a href='/linkedhashes.php?g=$gameID'>Supported Game Files</a>";
+        echo " <a href='/viewtopic.php?t=$forumTopicID'>Official Forum Topic</a>";
```

Comment:
This link should be hidden if the game doesn't have an official forum topic.
```
select ID from GameData where ForumTopicID is null;
```

Reply:
Good catch, fixed.